CHANGES.rst

```diff
@@ -7,10 +7,67 @@ Version 5.0.0
 
 Released: XXXX-XX-XX
 
-Changes
--------
+Fixes
+-----
+
+- Fix ValueError from unregistering services when application stops (#345)
+- Fix the MOTD example (#319)
+- Fix the Hello_World example (#318)
+- Fix the attractors tasks application example (#317)
+- Make TasksApplication.gui expect an IGUI interface, not a GUI instance (#301)
+
+Features
+--------
+
+- Re-export CorePlugin in envisage.api (#332)
+- Create and fill plugin subpackage api modules (#323)
+- Add relevant classes to envisage.ui.tasks.api (#322)
+
```

Comment:
I think it is typical for "Features" to go before "Fixes"?  At least that's the ordering used for Version 4.9.0